core/src/main/scala/cats/Bifoldable.scala

```diff
@@ -0,0 +1,46 @@
+package cats
+
+/**
+ * A type class abstracting over types that give rise to two independent [[cats.Foldable]]s.
+ */
+trait Bifoldable[F[_, _]] extends Serializable { self =>
```

Comment:
Hmm I see why we would want to do that, but it doesn't seem like we do it in other type classes.. do we?